### PR TITLE
Reduced log verbosity for pieces that are stable

### PR DESCRIPTION
**Pull request by davidzhao**

### CODE REVIEW

**Review by cnderrauber — APPROVED** [review]

File: pkg/rtc/signalhandler.go [hunk 1]
```diff
@@ -57,8 +57,6 @@ func HandleParticipantSignal(room types.Room, participant types.LocalParticipant
 				pLogger.Errorw("failed to update subscribed track settings", err, "trackID", sid)
 				continue
 			}
-
-			pLogger.Infow("updated subscribed track settings", "trackID", sid, "settings", msg.TrackSetting)
 		}
```
Comment (boks1971) [inline pkg/rtc/signalhandler.go:60]: Probably leave this in at Infow till we have signal deduper full confidence?

File: pkg/rtc/subscribedtrack.go [hunk 2]
```diff
@@ -120,6 +127,11 @@ func (t *SubscribedTrack) SetPublisherMuted(muted bool) {
 func (t *SubscribedTrack) UpdateSubscriberSettings(settings *livekit.UpdateTrackSettings) {
 	prevDisabled := t.subMuted.Swap(settings.Disabled)
 	t.settings.Store(settings)
+
+	if prevDisabled != settings.Disabled {
+		t.logger.Infow("updated subscribed track enabled", "enabled", !settings.Disabled)
```
Comment (boks1971) [inline pkg/rtc/subscribedtrack.go:132]: Oh you have it here

**Review by boks1971 — APPROVED** [review]

File: pkg/rtc/signalhandler.go (hunk 1 above)
Comment (davidzhao) [reply pkg/rtc/signalhandler.go:60]: I don't believe signal deduper is currently handling adaptive stream messages. These are being logged in SubscribedTrack when there's an actual change in layer or visibility. I thought that'd be sufficient. wdyt?

Comment (boks1971) [reply pkg/rtc/signalhandler.go:60]: Signal deduper does handle UodateTtackSettings messages. We get a lot of those duplicates too. As long as it is logged in SubscribedTrack, it should be fine.

**Pull request merged**